core/src/main/java/org/apache/druid/math/expr/Function.java

```diff
@@ -1910,16 +1910,10 @@ public String name()
     @Override
     public ExprEval apply(List<Expr> args, Expr.ObjectBinding bindings)
     {
-      Long left = args.get(0).eval(bindings).asLong();
-      Long right = args.get(1).eval(bindings).asLong();
+      long left = args.get(0).eval(bindings).asLong();
```

Comment:
Hmm, I actually think this should probably be checking if either result `isNumericNull` and returning a null result before calling `asLong`. Looking around there are ... other places in the functions that have this issue as well, so I don't think you need to fix here.